examples/RN.0.7x-cli/.yarnrc.yml

```
@@ -0,0 +1,11 @@
+cacheFolder: ../../../.yarn/cache
+
+nodeLinker: node-modules
+
+plugins:
+  - path: .yarn/plugins/@yarnpkg/plugin-version.cjs
+    spec: "@yarnpkg/plugin-version"
+
+yarnPath: .yarn/releases/yarn-3.6.4.cjs
+
+checksumBehavior: "ignore"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)